src/ucp/api/ucp_def.h

```diff
@@ -234,5 +234,15 @@ typedef void (*ucp_send_callback_t)(void *request, ucs_status_t status);
 typedef void (*ucp_tag_recv_callback_t)(void *request, ucs_status_t status,
                                         ucp_tag_recv_info_t *info);
 
+/**
+ * @ingroup UCP_WORKER
+ * @brief UCP worker wakeup events mask.
+ *
+ * The enumeration allows specifying which events are expected on wakeup
+ */
+typedef enum ucp_wakeup_event_types {
+    UCP_WAKEUP_TX_COMPLETION   = UCS_BIT(0), /**< TX-completions */
+    UCP_WAKEUP_RX_AM           = UCS_BIT(1)  /**< RX-completions */
```

Comment:
@evgeny-leksikov i think @shamisp means that UCP event names and documentation should be consistent with type of operations in UCP level.
for example:
```
UCP_WAKEUP_RMA             = UCS_BIT(0),  /**< Remote memory access send completion */
UCP_WAKEUP_AMO             = UCS_BIT(1),  /**< Atomic operation send completion */
UCP_WAKEUP_TAG_SEND    = UCS_BIT(2), /**< Tag send completion  */
UCP_WAKEUP_TAG_RECV    = UCS_BIT(3),  /**< Tag receive completion */
```